source/_integrations/vallox.markdown

```diff
@@ -67,6 +67,23 @@ automation:
 ```
 {% endraw %}
 
+In order to also update the input select in case some external event changes the Vallox profile (web interface, mechanical switch, reboot, etc...) you can use the following automation:
+
+{% raw %}
+```yaml
+automation:
+ - alias: Update vallox input_select
+  description: Update input_select when external event changes the profile
```

Comment:
This YAML is actually invalid, as it is wrongly indented.

Reply:
OK, should be valid now. I am a yaml noob to be honest, so thanks for the feedback.